usaspending_api/data_load/data_load_helpers.py

```diff
@@ -0,0 +1,32 @@
+import datetime
+
+subtier_agency_list = {}  # global variable, populated by fpds_loader
+
+
+def capitalize_if_string(val):
+    if isinstance(val, str):
+        return val.upper()
+    return val
+
+
+def false_if_null(val):
+    if val is None:
```

Comment:
I like my exception logic separate, but I'm a Java guy. If it's more pythonic in one line it's more pythonic in one line

Reply:
It can stay as-is. I should have been more clear by asking if the one-liner exhibits the same behavior or if there are falsy values you wish to be returned (link an empty string, empty list, ...)

I'm fuzzy on the details but I vaguely remember a previous conversation not too long ago about functions being too small

Reply:
Good thing it's three lines! On a serious note, I would defend a short function like this by the fact that it's being used in several places, as opposed to being a "helper" step in a bigger function. Thanks for keeping me honest to my own rules.

Reply:
Are there falsy values you wish to be returned or should it always return a boolean?

Reply:
This should always return a boolean. Is there a better way I should be doing this?

Reply:
`return True if val else False`

Reply:
Turns out that most of these values are coming through as strings representing bools. I loosened the requirements to just let Postgres translate them into and out of boolean.